src/main/java/com/palantir/websecurity/filters/WebSecurityHeaderInjector.java

```diff
@@ -22,7 +22,8 @@
  */
 public final class WebSecurityHeaderInjector {
 
-    public static final String DEFAULT_CONTENT_SECURITY_POLICY = "default-src 'self'; style-src 'self' 'unsafe-inline'";
+    public static final String DEFAULT_CONTENT_SECURITY_POLICY =
+            "default-src 'self'; style-src 'self' 'unsafe-inline'; frame-ancestors 'self';";
```

Comment:
they aren't supported at this time, so they are ignored. when they are supported, XFrameOptions will be ignored, meaning this would be insecure. having to time the swap is not advisable, instead having both headers prevents any lapse in frame security settings being applied. see the issue for more of a conversation